pkg/adapter/adapter.go

```diff
@@ -152,20 +153,29 @@ func (a *mongoDbAdapter) makeCloudEvent(data bson.M) (*cloudevents.Event, error)
 	// Create Event.
 	event := cloudevents.NewEvent(cloudevents.VersionV1)
 
-	// Set cloud event specs and attributes. TODO: issue #43
+	// Set cloud event specs and attributes.
 	// 		ID     -> id of mongo change object
 	// 		Source -> Source environment variable.
-	// 		Type   -> type of change either insert, delete or update.
+	// 		Type   -> type of change either inserted, deleted or updated.
 	//		Data   -> data payload containing either id only for
 	//                deletion or full object for other changes.
```

Comment:
I think I'd rather remove all this comment. Is clearer if you just read the code

Reply:
True!